README.md

```diff
@@ -22,80 +22,47 @@ Much like the contents of a container image need to be pulled and unpacked in or
 In this sense, Bundle is a generalization of the image concept, and can be used to represent any type of content.
 
 `Bundles` do nothing on their own - they require a `Provisioner` to unpack and make their content available in-cluster. They can be
-unpacked to any arbitrary storage medium such as a PersistentVolume or ConfigMap. Each `Bundle` has an associated `spec.class` field which 
+unpacked to any arbitrary storage medium such as a PersistentVolume or ConfigMap. Each `Bundle` has an associated `spec.provisionerClassName` field which 
 indicates the `Provisioner` that should be watching and unpacking that particular bundle type. 
 
 Example Bundle
 ```yaml
 apiVersion: core.rukpak.io/v1alpha1
 kind: Bundle
-metadata: 
-  name: example-operator.v0.9.3
+metadata:
+  name: combo-0.0.1
 spec:
-  class: rukpack.io/k8s
-  refs:
-  - file://content
-  volumeMounts:
-  - mountPath: /content
-    configMap:
-      name: local 
-      namespace: plumbus
+  image: quay.io/tyslaton/combo-plain-bundle:v0.0.1
+  provisionerClassName: core.rukpak.io/plain
 ```
 
-### Instance 
-The `Instance` API points to a Bundle and indicates that it should be “active”. This includes pivoting from older versions of an active bundle. 
-`Instance` may also include an embedded spec for a desired Bundle.
+### BundleInstance 
+The `BundleInstance` API points to a Bundle and indicates that it should be “active”. This includes pivoting from older versions of an active bundle. 
+`BundleInstance` may also include an embedded spec for a desired Bundle.
```

Comment:
Is it worth mentioning that the embedded Bundle spec won't be present until a future release?

Reply:
maybe it makes the most sense to remove that for now and then add it back once the functionality becomes available -- do we have a ticket tracking that feature?

Reply:
Yeah that sounds reasonable to me - I don't think we have a tracking ticket anywhere.

Reply:
I just created https://github.com/operator-framework/rukpak/issues/73 btw before we lose this thread in the sea of comments.